### PR TITLE
Update protocol to support pluggable devices

### PR DESCRIPTION
**Pull request by fstasi**
Breaking: This PR addresses compatibility issues with the new pluggable monitor implemented in Arduino IDE2.

In order to support pluggable devices, the Protocol must change accordingly:
`pluggableMonitorSettings` is used for actual monitor/CLI 
`monitorUISettings` is used for UI only information

### CODE REVIEW

**Review by davegarthsimpson — APPROVED** [review]

File: README.md
Comment (AlbyIanna) [inline README.md:85]:
```suggestion
Settings changes, sent and received in the Serial Plotter App, must have the following object structure
```

File: README.md
Comment (AlbyIanna) [inline README.md:94]:
```suggestion
That means a Setting Message can contain `pluggableMonitorSettings` and/or `monitorUISettings`.
```

**Review by AlbyIanna — APPROVED** [review]

File: package.json
Comment (AlbyIanna) [inline package.json:3]:
```suggestion
  "version": "0.1.0",
```

Comment (AlbyIanna) [reply package.json:3]: We significantly changed the protocol messages, let's bump the minor.

**Comment by AlbyIanna** [comment]
@davegarthsimpson would you please have a look at the comments I made? After that, let's merge this PR right away if no other change is needed.

**Pull request merged**
